api/app/Events/ApplicationSubmitted.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Events;
+
+use App\Models\PoolCandidate;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Broadcasting\PresenceChannel;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Log;
```

Comment:
Some unused references

Reply:
Removed in [remove channels and unused imports](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3938/commits/970f5f898be17ef2a7cfbc4bacad4d332ea154b4)